src/commands/Minion/cast.ts

```diff
@@ -89,6 +99,14 @@ export default class extends BotCommand {
 			])
 		);
 
+		if (spell.gpCost) {
+			const gpCost = spell.gpCost * quantity;
+			if (gpCost < userGP) {
+				return msg.send(`You need ${gpCost} GP to create ${quantity} planks.`);
```

Comment:
Probably better to write just "to cast this spell quantity times" because it assumes every spell that requires gp is one that makes planks, unless that is actually true

Reply:
gp cost only applies to plank make, no cost on tan leather